restful-ws/3.0/_index.md

```diff
@@ -11,7 +11,8 @@ following the Representational State Transfer (REST) architectural pattern.
 * [Jakarta RESTful Web Services 3.0 Specification Document](./jakarta-restful-ws-spec-3.0.pdf) (PDF)
 * [Jakarta RESTful Web Services 3.0 Specification Document](./jakarta-restful-ws-spec-3.0.html) (HTML)
 * [Jakarta RESTful Web Services 3.0 Javadoc](./apidocs)
-* [Jakarta RESTful Web Services 3.0 TCK](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip)([sig](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip.sig),[sha](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* [Jakarta RESTful Web Services 3.0.1 TCK](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip)  ([sig](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip.sig),  [sha](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
@edbratt It looks like there was a 3.0.0 TCK...  Was it never used for compatibility testing?  Was there something wrong with it?  I would have expected a top entry for 3.0.0 and then two sub-bullets for 3.0.1 and the new 3.0.2 TCK.

Reply:
PR log review shows:

[Update Platform TCK 9.0.2 + Jakarta RESTful Web Services 3.0.1 for TCK challenge #330](https://github.com/jakartaee/specifications/pull/330) -- Feb 10 **replaced** TCK links: Jakarta EE Platform TCK 9.0.1 with 9.0.2 and Restful Web Service TCK link 3.0.0 with 3.0.1. This was in response to an accepted test challenge ([JAXRS-API #937](https://github.com/eclipse-ee4j/jaxrs-api/issues/937)). 

Strictly speaking, this PR should have ADDED the new TCKs, but we weren't clear on that. I did not attempt to rectify these prior decisions. We certainly could if you and/or the committee things that would be helpful.

Reply:
All that I am indicating is that we should list the 3.0.0 TCK at the top level, with the 3.0.1 and 3.0.2 TCKs as sub-bullets.  I think that completes the picture for the TCKs, doesn't it?  Just pretending that the 3.0.0 TCK never existed doesn't seem right.

Reply:
Please see the latest commit.

Reply:
I added a note to https://github.com/eclipse-ee4j/jakartaee-tck/wiki/TCK-Challenge-processing-status pointing back to these comments about keeping the original TCK zip links and just adding updated (with excluded tests) as a sub-bullet item.